conformance/01_pull_test.go

```diff
@@ -0,0 +1,137 @@
+package conformance
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/bloodorangeio/reggie"
+	g "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var test01Pull = func() {
+	g.Context("Pull", func() {
+
+		var tag string
+
+		g.Context("Setup", func() {
+			g.Specify("Populate registry with test blob", func() {
+				RunOnlyIf(runPullSetup)
+				req := client.NewRequest(reggie.POST, "/v2/<name>/blobs/uploads/")
+				resp, _ := client.Do(req)
```

Comment:
I think that all the setup/teardown steps should in fact run some Expect() 's. It's ok if they are counted against skipped tests, but we should expose any issues with setup/teardown steps and consider it a failure. Perhaps its more forgiving (201 vs 202 etc, just check for success)

Example Quay.io - "Populate with test manifest" shows green, then they fail with 404. The upload actually fails if you open the logs due to manifest validations: https://oci-distribution-conformance-results.s3.amazonaws.com/quay-split-tests.html

Reply:
done, thanks